shiny/render/_dataframe.py

```diff
@@ -258,29 +402,334 @@ class data_frame(Renderer[DataFrameResult]):
       objects you can return from the rendering function to specify options.
     """
 
-    # `<data_frame_renderer>.input_selected_rows()` method, where `<data_frame_renderer>` is the render function name that corresponds with the `id=` used in :func:`~shiny.ui.outout_data_frame`. Internally, this method retrieves the selected row value from session's `input.<id>_selected_rows()` value. The value returned will be `None` if no rows
+    _session: Session | None  # Do not use. Use `_get_session()` instead
+
+    _value: reactive.Value[DataFrameResult | None]
+    # _data: reactive.Value[pd.DataFrame | None]
+
+    _cell_update_fn: CellUpdateFn
+    _cells_update_fn: CellsUpdateFn
+
+    cell_patches: reactive.Value[list[CellPatch]]
+
+    data: reactive.Calc_[pd.DataFrame]
+    """
+    Reactive value of the data frame's output data.
+    """
+    data_patched: reactive.Calc_[pd.DataFrame]
+    """
+    Reactive value of the data frame's edited output data.
+    """
```

Comment:
Should explain that:

- the first of these is the original object, so if mutated, it will modify the original data
- the second is a shallow copy of the original columns; modifying the columns _may_ modify the original data

at least, that is my understanding.